### PR TITLE
Added eslint to travis run

### PR DESCRIPTION
**Pull request by apepper**
This PR adds `eslint` as its own `npm run` command. So `npm run eslint` will check all files against eslint. Webpack builds are not affected. By this we should have the best of two worlds:

1. Easy onboarding, since webpack builds do not fail or report warnings from eslint.
2. Get a consistent code base, since travis will fail, if eslint has a warning or an error.

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    "ecmaVersion": 8,
+    "sourceType": "module",
+  },
+  "extends": [
+    "eslint:recommended",
+    "airbnb/base",
+    "plugin:react/recommended",
+  ],
+  "globals": {
+    "Blob": true,
```
Comment (dcsaszar) [inline .eslintrc]: can be reduced to:
```
  "globals": {
    "document": true,
    "React": true,
    "Scrivito": true,
    "window": true,
  },
```

File: .eslintrc [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    "curly": ["error", "all"],
+    "default-case": 0,
+    "import/no-amd": 1,
+    "import/no-commonjs": 1,
+    "import/no-extraneous-dependencies": "off",
+    "import/no-nodejs-modules": 1,
+    "function-paren-newline": 0,
+    "new-cap": 0,
+    "no-debugger": 1,
+    "no-new": 0,
```
Comment (dcsaszar) [inline .eslintrc]: Not needed?

File: .eslintrc [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    }],
+    "consistent-return": 0,
+    "curly": ["error", "all"],
+    "default-case": 0,
+    "import/no-amd": 1,
+    "import/no-commonjs": 1,
+    "import/no-extraneous-dependencies": "off",
+    "import/no-nodejs-modules": 1,
+    "function-paren-newline": 0,
+    "new-cap": 0,
```
Comment (dcsaszar) [inline .eslintrc]: Not needed?

File: .eslintrc [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+    "no-debugger": 1,
+    "no-new": 0,
+    "no-param-reassign": [2, { "props": false }],
+    "no-proto": 0,
```
Comment (dcsaszar) [inline .eslintrc]: Not needed?

File: .eslintrc [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+    "object-curly-newline": 0,
+    "prefer-destructuring": 0,
+    "react/display-name": 0,
+    "react/jsx-curly-spacing": [2, "always"],
+    "react/jsx-no-undef": ["error", { "allowGlobals": true }],
+    "react/no-danger": 0,
+    "react/no-direct-mutation-state": 0,
+    "react/no-is-mounted": 0,
+    "react/prop-types": 0,
+    "vars-on-top": 0,
```
Comment (dcsaszar) [inline .eslintrc]: Not needed?

Note: I didn't check all options.

File: .eslintrc [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+    "wrap-iife": ["error", "any"],
+  },
+  overrides: [
```
Comment (dcsaszar) [inline .eslintrc]: A shorter alternative would be allowing `console`, but instructing uglifyjs to remove these statements.

File: .travis.yml [hunk 7]
```diff
@@ -2,7 +2,7 @@
 language: node_js
 node_js: 8
 sudo: false
-script: npm run build
+script: npm run build && npm run eslint
```
Comment (dcsaszar) [inline .travis.yml]: * Splitting this to two jobs would reduce the feedback time.
* Having a separate package.json would massively speed up the lint test.
* As a simple improvement we should swap the two tests

**Comment by apepper** [comment]
REVIEW (d26cfff)

Cleaned up `globals` and splited tasks on travis into two.

File: .travis.yml [hunk 8]
```diff
@@ -2,9 +2,13 @@
...
 sudo: false
-script: npm run build && npm run eslint
 env:
   SCRIVITO_TENANT: "c1bd936ef25b6dcccfd7b72b71fe969d"
 cache:
   directories:
     - "node_modules"
+env:
+  matrix:
+    - TASK=build
```
Comment (dcsaszar) [inline .travis.yml]: There still is some confusion about this. (https://github.com/travis-ci/docs-travis-ci-com/issues/1500)
IMO these are `jobs`. But that's just an opinion.

**Pull request merged**